queries/cdmq/cdm.js

```diff
@@ -1637,6 +1644,11 @@ getMetricDataFromIdsSets = function (url, sets, metricGroupIdsByLabelSets) {
         var k;
         for (k = 2; k < 4; k++) {
           //for my $j (@{ $$resp_ref{'responses'}[$count + $k]{'hits'}{'hits'} }) {
+          if (data.responses[count + k].hits.total.value !== data.responses[count + k].hits.hits.length) {
+            console.log("WARNING! data.responses[" + (count + k) + "].hits.total.value (" + data.responses[count + k].hits.total.value +
+                        ") and data.responses[" + (count + k) + "].hits.hits.length (" + data.responses[count + k].hits.hits.length +
+                        ") are not equal, which means the retured data is probably incomplete");
+          }
```

Comment:
Ditto. error handling?

Reply:
The only thing we can do is exit, but I thought I would set a warning first and see if anyone comes across this.